_includes/current_sponsors.html

```diff
@@ -1,11 +0,0 @@
-<div id="current-sponsors-container">
```

Comment:
The file did get deleted....someone set it up as a partial for use in the sponsor_show.html and there was no clear reason why that specific chunk should be made a partial.  So I rolled it up into the sponsor_show.html page and deleted current_sponsors.html.

As far as my other branch, _that_ branch should be integrated first.  I tested the merge with the branch integration in that order and it works fine on my localhost.